docs/integrations/sources/mssql.md

```diff
@@ -61,19 +9,30 @@ Please see [this issue](https://github.com/airbytehq/airbyte/issues/4270) for de
 | Replicate Incremental Deletes | Yes |  |
 | CDC (Change Data Capture) | Yes |  |
 | SSL Support | Yes |  |
-| SSH Tunnel Connection | Coming soon |  |
+| SSH Tunnel Connection | Yes |  |
 | Namespaces | Yes | Enabled by default |
 
-## Getting started
+The MSSQL source does not alter the schema present in your database. Depending on the destination connected to this source, however, the schema may be altered. See the destination's documentation for more details.
 
-### Requirements
+## Troubleshooting
+
+#### Issue: Connector provides wrong values for some data types.
+Please see [this issue](https://github.com/airbytehq/airbyte/issues/4270) for description of unexpected behaviour for certain datatypes.
+
+#### Issue: Managed versions of MSSQL
+We are unsure about the nuances behind managed versions of MSSQL, so if you run into any, please update this document!
```

Comment:
We shouldn't publish docs that make such a blanket statement. As a managed MSSQL user reading this, this just decreases my confidence in this connector. What made you add this line? We should either make this very specific (say exactly what the failure modes are) or remove it

Reply:
Yeah, fair question.

This section used to be in our MSSQL docs:

```
#### Setting up CDC on managed versions of SQL Server

We readily welcome [contributions to our docs](https://github.com/airbytehq/airbyte/tree/master/docs) providing setup
instructions. Please consider contributing to expand our docs!
```

so I just turned that into a Troubleshooting thing. I'll just take it out honestly.